PieceReference.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.groupproject
+
+data class PieceReference(
+    val Rook: String,
+    val Knight: String,
+    val Bishop: String,
+    val King: String,
+    val Queen: String,
+    val Pawn: String,
+)
```

Comment:
What is the point of this class?

Reply:
This is a reference point for the chess pieces for use in the XML files